client/sites/components/sites-dataviews/dataview-style.scss

```diff
@@ -3,15 +3,15 @@
 @import "@wordpress/dataviews/build-style/style.css";
 
 .sites-dataviews {
-	.dataviews-view-list {
-		// DataView overrides: Ideally instead of reusing the site name full field
-		// We should be setting a media field and a primary field.
-		// Or update Core to hide them when not specified.
-		.dataviews-view-list__media-wrapper {
-			display: none;
-		}
-		.dataviews-view-list__primary-field {
-			display: none;
+	table.dataviews-view-table .dataviews-view-table__row td {
+		// dataviews-override: We combine the icon and site title fields to form the horizontal "site" field.
+		// Without this rule the combined field receives justify-content: space-between but we want the icon to be aligned with the text.
+		// This can be removed when https://github.com/WordPress/gutenberg/pull/67226 / Gutenberg 19.8 lands in calypso.
```

Comment:
![badge](https://img.shields.io/badge/review-nits-green.svg)

```suggestion
		// TODO: This can be removed when https://github.com/WordPress/gutenberg/pull/67226 / Gutenberg 19.8 lands in calypso.
```